php/commands/sites.php

```diff
@@ -341,8 +341,8 @@ public function mass_update($args, $assoc_args) {
         $update = $noupdatedb ? false : true;
         $xoption = Input::optional($assoc_args, 'xoption', 'theirs');
         if (!$report) {
-          $confirmed = Input::yesno("Apply upstream updatefs to %s ( run update.php:%s, xoption:%s ) ", array($site->getName(), var_export($update,1), var_export($xoption,1)));
-          if( !$confirmed ) continue; // Suer says No, go back to start.
+          $confirmed = Input::yesno("Apply upstream updates to %s ( run update.php:%s, xoption:%s ) ", array($site->getName(), var_export($update,1), var_export($xoption,1)));
+          if( !$confirmed ) continue; // User says No, go back to start.
```

Comment:
Thanks @TeslaDethray. Which style guide are we following? I see that CONTRIBUTING.md says to follow the WordPress PHP Coding Standards, but on https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage I see

> Put spaces on both sides of the opening and closing parenthesis of if, elseif, foreach, for, and switch blocks.